Review.md

```diff
@@ -1,6 +1,14 @@
 # Please answer the following questions
 
 1.  In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+`describe()` helps to break down the test into smaller units (e.g. a counter) while `it()` helps to describe each inidividual test (i.e. counter should start counting from 0).
+
 2.  What is the point of `Test Driven Development`? What do you think about this approach?
+TDD helps developers to think more clearly in terms of the requirements from differenmt stakeholders before diving into writing codes. On top of this, it helps to maintain a project in a scalable way and communication costs between teams and members. In my opinion, TDD is pretty beneficial for inidividual and also the team, although it has to take more time and effort to create and maintain those tests.
+
 3.  What are `mocks`? What are a good use cases for them?
+mocks
+The main goal of mocking is to separate the code we do not control with the code we do control, which helps to isolate test subjects by replacing dependecies. 
+
 4.  Mention three types of automated tests.
+UI, unit and APIs.
```

Comment:
Unit testing, Integration testing and end to end testing are the three automated tests